### PR TITLE
LRAC-9366

### PR DESCRIPTION
**Pull request by catarineaguiar**

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
ci:test:analytics-cloud:nocompile

**Comment by catarineaguiar** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
ci:test:analytics-cloud:nocompile

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase [hunk 1]
```diff
@@ -364,11 +364,15 @@ definition {
...
+			Pause(locator1 = "5000");
+
 			Navigator.gotoSitePage(
 				pageName = "AC Page",
 				siteName = "Site Name");
 
 			DMNavigator.gotoDocumentPG(dmDocumentTitle = "New Document");
+
+			Pause(locator1 = "5000");
 		}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase:376]: Hi @catarineaguiar, I think you could put this in `for`

**Pull request closed without merging**